### DISCUSSION TITLE
at row 164 ---> for author in authors <---  here indent error

### DISCUSSION THREAD

**Discussion by harunfr** [mdn/mdn-community#571] [Content] [answered]
at row 164 ---> for author in authors <---  here indent error

**Comment by hamishwillee** [comment, marked as answer]
Howdy, the right place to create an issue is in the Issues section https://github.com/mdn/content/issues, providing all the information that is requested. That allows us to easily find the problem section and know why you think it is a problem. For example, I would not necessarily consider indentation wrong if the code still worked, even if it "looked wrong".

Anyway, I'm guessing you mean this line, which looks find to me: https://github.com/mdn/content/blob/main/files/en-us/learn/server-side/express_nodejs/forms/create_book_form/index.html#L164

**Reply by hamishwillee** [reply]
PS I have marked this as an answer ^^^. If there is really an issue, please create one with enough info for us to debug. I don't plan to continue this as a discussion, because it is hard to manage "completion" tracking.

**Comment by pepelsbey** [comment]
Closing as answered/settled.
